Source/Core/Common/Analytics.cpp

```diff
@@ -134,13 +134,15 @@ AnalyticsReporter::~AnalyticsReporter()
 
 void AnalyticsReporter::Send(AnalyticsReportBuilder&& report)
 {
+#ifdef USE_ANALYTICS
```

Comment:
So? What if someone sets it to 0 in the future? You need to take this into account.